test/SILGen/enum_raw_representable_available.swift

```diff
@@ -0,0 +1,67 @@
+// RUN: %target-typecheck-verify-swift
+// RUN: %target-swift-emit-silgen -emit-sorted-sil %s | %FileCheck %s
+// RUN: %target-swift-emit-silgen -emit-sorted-sil -enable-resilience %s | %FileCheck %s
+
+// Really just requires a platform with meaningful runtime #available() tests.
+// REQUIRES: objc_interop
```

Comment:
I think it would be fine to limit this to "just macOS" or something, rather than get bitten when Apple introduces a new platform some day. `REQUIRES: OS=macosx`